inc/sitemaps/class-post-type-sitemap-provider.php

```diff
@@ -283,6 +283,8 @@ public function save_post( $post_id ) {
 	 */
 	public function is_valid_post_type( $post_type ) {
 
+		global $wp_post_types;
```

Comment:
I had profiled performance difference, on 900 calls relative slowdown from API call for method is 300+%, but in absolute numbers that's only about 12ms extra (note that this in context of pretty powerful desktop PC).

If you insist on using API here I guess I am fine with these numbers.